tools/wireshark_plugin/README.md

```diff
@@ -0,0 +1,13 @@
+# vsomeip-dissector
+Wireshark dissector for vSomeip internal communication via TCP
+
+## How To Use
+
+1. Place `vsomeip-dissector.lua` file in `~/.config/wireshark/plugins/vsomeip/vsomeip-dissector.lua`
+   (create `plugins` directory if it doesn't exist)
+2. In wireshark go to `Analyze` > `Reload Lua Plugins`
+3. In wireshark go to `Analyze` > `Enable Protocols` and search for `vsomeip` and enable it
```

Comment:
I followed these steps, is there any configuration for it?  I see that `vsomeip` is now a protocol in my display filters, but I don't see it in the protocol list in the list of protocols (along sign SOME/IP and the 1000 other ones.)

Also, when I filter on vsomeip, I get no results.  With SOME/IP I had to configure the port.  When I forcefully tell it to decode as vsomeip (adding `decode_as_entry: tcp.port,30510,SOME/IP,VSOMEIP` in `.config/wireshark/decode_as_entries`) then nothing shows up.

Reply:
Hello @kheaactua 
No extra configurations should be needed.
This plugin will only work for the vsomeip internal communication via tcp.
External someip messages will still be dissected by the default someip protocol on wireshark.

for the internal vsomeip communication via tcp, it should look something like this:
![image](https://github.com/user-attachments/assets/59409a46-faf8-456a-8724-f524a79c0997)
![image](https://github.com/user-attachments/assets/1fad61f0-5653-4913-a22e-e6464708b784)

Reply:
Oh, that explains why I didn't see anything, the pcap file I used only had external TCP (all local was done with UDS.)

This is pretty cool.

Thanks for the info!